3. Installation/3. Docker Containers/systemd/README.md

```diff
@@ -4,20 +4,101 @@ Deploy [Rocket.Chat](https://github.com/RocketChat/Rocket.Chat) to Linux that ru
 
 ## Prerequisite
 
-You need to have [docker](https://docs.docker.com/linux/started/) and [docker-compose](http://docs.docker.com/compose/) installed.
+You need to have [docker](https://docs.docker.com/linux/started/) installed.
 
 ## How to run Rocket.Chat on systemd.
 
+Create three systemd files. One for the database, one for the database replica and one for rocketchat.
+
+mongo.service:
+```
+[Unit]
+Description=mongo
+Requires=docker.service
+After=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+User=dockeruser
+Restart=always
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill mongo
```

Comment:
I am using this in production and it is killing and removing this everytime it is recreated. The persistent data is in the volume. I also update the mongo image if something is pushed (i think it is the same way for docker-compose if i pull, and make up -d)